### ISSUE TITLE
Safari extension is not able to install.

### ISSUE THREAD

**Issue by D-elias** [YePpHa/YouTubeCenter#2060]
Every time it gives me the error "An error occurred while installing the extension “YouTube Center Developer Build” when I try to install the Safari: YouTubeCenter.safariextz.

**Comment by Phosphor303** [comment]
Confirmed for Safari 7.1.8 on OS X 10.9.5

**Comment by qwertimus** [comment]
Confirmed for Safari 9.0 on El Capitan DP8

**Comment by Crazor** [comment]
confirmed on El Capitan with Safari 9.0 build 11601.1.56

Console shows the following errors:
```17.09.15 11:37:22,000 kernel[0]: Sandbox: com.apple.WebKit(1065) deny(1) file-read-data /Applications/Safari.app```
```17.09.15 11:37:22,000 kernel[0]: Sandbox: com.apple.WebKit(1065) deny(1) file-read-data /Applications/Safari.app/Contents
``

**Comment by TheTruthSentMe** [comment]
Confirmed on Yosemite 10.10.5 (14F27) with Safari 8.0.8 (10600.8.9).
Currently running YTC build 525. Would like to update.

**Comment by DreamSeason** [comment]
It has been around for quite long. User TamperMonkey instead.:D

**Comment by qwertimus** [comment]
I agree with @DreamSeason. I've switched to the userscript version through TamperMonkey as well now. Works exactly the same, super happy YTC's working properly again.

**Comment by LiamBM** [comment]
Anyone care to explain how TamperMonkey works? I'm not the best with computers, haha.

**Comment by TheTruthSentMe** [comment]
1. Download and install TamperMonkey extension from http://tampermonkey.net
2. Open YTC UserScript at https://github.com/YePpHa/YouTubeCenter/blob/master/dist/YouTubeCenter.user.js
3. TamperMonkey should now have installed YTC on its own

**Comment by LiamBM** [comment]
@TheTruthSentMe, Thanks, works great. The only thing that still doesn't seem to work on safari, is the auto pause. Not a big deal though.
